encrypted-media-respec.html

```diff
@@ -1498,17 +1498,19 @@
             The time represented by the <a def-id="expiration"></a> attribute MUST be earlier than the current time.
             All other keys in the session MUST have this status.
           </dd>
+          <dt>output-restricted</dt>
+          <dd>
+            There are output restrictions associated with the key which cannot currently be met.
+            The application should avoid using streams which will trigger the output restrictions associated
+            with the key.
```

Comment:
We should specify what will happen. We could either add that to the end of this sentence (i.e. "... to avoid being unable to encrypt it") or in between these two sentences like I have proposed for "output-downscaled" in my first comment above.